docs/src/pages/components/avatars/avatars.md

```diff
@@ -24,3 +24,9 @@ Avatars containing simple characters can be created by passing your string as `c
 Icon avatars are created by passing an icon as `children`.
 
 {{"demo": "pages/components/avatars/IconAvatars.js"}}
+
+## Variants
+
+Do you need square or rounded avatars? Use the `variant` prop.
```

Comment:
We should add the third variant here?

Reply:
the third variant you refer to is the default one, right? circle?

Reply:
Otherwise write 'Do you need square or rounded avatars instead?" which implies the circle is default?

Reply:
@mattdotam this sounds great.